Model/Api/Data/AutomatedTesting/OrderItem.php

```diff
@@ -0,0 +1,236 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2017-2021 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Model\Api\Data\AutomatedTesting;
+
+class OrderItem implements \JsonSerializable
+{
+    /**
+     * @var string
+     */
+    private $productName;
+
+    /**
+     * @var string
+     */
+    private $productSku;
```

Comment:
We need to have URL to product page as well

Reply:
sounds good.